README.md

```diff
@@ -54,16 +54,30 @@ For Debian-based systems:
 
 ### Build a tar archive
 
+First clone the repository and choose the right tag or branch
+
 ```sh
 git clone --recurse-submodules https://github.com/ungoogled-software/ungoogled-chromium-portablelinux.git
+cd ungoogled-chromium-portablelinux
 # Replace TAG_OR_BRANCH_HERE with a tag or branch name
 git checkout --recurse-submodules TAG_OR_BRANCH_HERE
+```
+
+Then run a normal build
+
+```sh
 # Use "export ..." for AR, NM, CC, CXX, or others to specify the compiler to use
 # It defaults to LLVM tools. See build.sh for more details
 ./build.sh
 ./package.sh
 ```
 
+Or a docker build
+
+```sh
+./docker-build
```

Comment:
Typo

Reply:
Got it didn't really know how to say it, I clarified it a bit

Reply:
Thanks, but is the command `./docker-build` not a typo?

Reply:
you are right, I feel dumb for having missed that